js/main.js

```diff
@@ -1,9 +1,18 @@
 // main.js
 // Contains Javascript functions.....
 
-// Easter egg
-var rawr = function() {
-  document.getElementsByClassName('container')[0].style.paddingTop = 0;
-  document.getElementsByClassName('banner')[0].style.display = "none";
-  document.getElementsByClassName('soon')[0].innerHTML = "<img height='200px' src='images/rawrburt.png'>";
-}
+"use strict";
+
+(function() {
+  var soon = document.getElementsByClassName('soon')[0];
+
+  // Easter egg
+  document.getElementById("rawr").onclick = function() {
+      soon.innerHTML = "";
+      document.getElementsByClassName('banner')[0].style.display = "none";

```

Comment:
If you are grabbing an entire class of elements but only using the first element (or only have one element per class) you should consider just using an ID for the handle and styling.